src/TextArea/TextArea.tsx

```diff
@@ -31,6 +31,9 @@ export class TextArea extends React.Component {
     this.onBlur = this.onBlur.bind(this);
   }
 
+  // eslint-disable-next-line react/sort-comp
```

Comment:
eslint is reporting the following error:
`textAreaEl should be placed after onBlureslint(react/sort-comp)`
Which doesn't make any sense to me, since it's saying it should be placed between two instance properties. The error I got in other files told me to place the instance property after the last static property. Seems like a bug.

Reply:
Got it!